scripts/core/interactive-article-actions-panel/actions/publish-tab.tsx

```diff
@@ -159,13 +160,26 @@ export class PublishTab extends React.PureComponent<IProps, IState> {
                     <div style={style}>
                         <div>
                             {
-                                publishFromEnabled && (
+                                true && (
                                     <ToggleBox title={gettext('From')} initiallyOpen>
                                         <DestinationSelect
                                             value={this.state.selectedDestination}
                                             onChange={(value) => {
                                                 this.setState({
                                                     selectedDestination: value,
+                                                }, () => {
```

Comment:
It would be better to decouple UI from this new saving functionality. For example, could you do the saving in `componentDidUpdate`?

Reply:
That would be unnecessary updates. If we do it in the componentDidUpdate we'd have to go through all values no matter which one is changed. If we don't change it and handle it as I've done that wouldn't be the case. What I mean is that over all it seems like it would bring unnecessary complexity.

Reply:
> That would bring unnecessary updates

if you add an `if` to check for it, there won't be unnecessary updates.

It's computationally cheap to run a few extra loops, so if we can trade it for better code structure(decoupling UI from saving logic) I'd do that. See if it's possible to do it and keep code clean and readable. If not, I'm open to leaving it as is.